lib/pages/chat/events/message.dart

```diff
@@ -683,26 +683,26 @@ class Message extends StatelessWidget {
     );
   }
 
-  // Check if the sender of the current event is the same as the previous event.
-  bool isSameSender(Event? previousEvent, Event currentEvent) {
-    // If the previous event is null, it is assumed that the message is the newest.
-    if (previousEvent == null) {
+  // Check if the sender of the current event is the same as the compared event.
+  bool isSameSender(Event? comparedEvent, Event currentEvent) {
+    // If the compared event is null, it is assumed that the message is the newest.
+    if (comparedEvent == null) {
       return true;
     }
 
-    final isPreviousEventMessage = {
+    final isPreviousOrNextEventMessage = {
       EventTypes.Message,
       EventTypes.Sticker,
       EventTypes.Encrypted,
       EventTypes.Redaction,
-    }.contains(previousEvent.type);
+    }.contains(comparedEvent.type);
 
     // Ignoring events that are not messages, stickers, encrypted or redaction.
-    if (!isPreviousEventMessage) {
+    if (!isPreviousOrNextEventMessage) {
       return true;
     }
 
-    return previousEvent.senderId != currentEvent.senderId;
+    return currentEvent.senderId != comparedEvent.senderId;
```

Comment:
why we need to change it?

Reply:
I prefer to refactor this one so we can use it with previousEvent and nextEvent, didn't want to do 2 sames methods with exact same code